src/content/docs/en/guides/content-collections.mdx

```diff
@@ -410,236 +391,126 @@ const englishDocsEntries = await getCollection('docs', ({ id }) => {
+Much of the benefit of using collections comes from:
 
-## Working with dates in the frontmatter
+- Defining a common data shape to validate that an individual entry is "correct" or "complete", avoiding errors in production.
+- Content-focused APIs designed to make querying intuitive (e.g. `getCollection()` instead of `import.meta.glob()`) when importing and rendering content on your pages.
+- A [Loader API](/en/reference/loader-reference/) for retrieving your content that provides both built-in loaders and access to the low-level API. There are several third-party and community-built loaders available, and you can build your own custom loader to fetch data from anywhere.
+- Performance and scalability. The Content Layer API allows data to be cached between builds and is suitable for tens of thousands of content entries.
 
-Several date formats are possible in content collections, but your collection's schema must match the format used in your Markdown or MDX YAML frontmatter.
+[Define your data](#defining-collections) as a collection when:
 
-YAML uses the [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html) standard to express dates. Use the format `yyyy-mm-dd` (e.g. `2021-07-28`) along with a schema type of `z.date()`:
+- You have multiple files or data to organize that share the same overall structure (e.g. blog posts written in Markdown which all have the same frontmatter properties).
+- You have existing content stored remotely, such as in a CMS, and want to take advantage of the collections helper functions and Content Layer API instead of using `fetch()` or SDKs.
+- You need to fetch (tens of) thousands of related pieces of data, and need a querying and caching method that handles at scale.
 
-```markdown title="src/pages/posts/example-post.md"
----
-title: My Blog Post
-pubDate: 2021-07-08
----
-```
-
-The date format will be specified in UTC if a timezone is not provided. If you need to specify a timezone, you can use the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format. 
-
-```markdown title="src/pages/posts/example-post.md"
----
-title: My Blog Post
-pubDate: 2021-07-08T12:00:00-04:00
----
-```
-
-To render only the `YYYY-MM-DD` from the full UTC timestamp, use the JavaScript `slice` method to remove the timestamp:
-
-```astro title="src/layouts/ExampleLayout.astro"
----
-const { frontmatter } = Astro.props;
----
-<h1>{frontmatter.title}</h1>
-<p>{frontmatter.pubDate.toISOString().slice(0,10)}</p>
-```
-To see an example of using `toLocaleDateString` to format the day, month, and year instead, see the [`<FormattedDate />` component](https://github.com/withastro/astro/blob/latest/examples/blog/src/components/FormattedDate.astro) in the official Astro blog template. 
+### When not to create a collection
```

Comment:
Quick question/suggestion: Shouldn't we use the same heading level as `## When to create a collection`? When I saw this section nested inside the previous one [in the table of contents](https://5-0-0-beta.docs.astro.build/en/guides/content-collections/), it seemed strange to me.

```suggestion
## When not to create a collection
```

Reply:
My intention was it's part of the section, yes! It's the sub-section!

Reply:
Ok, if it's intentional it's good for me! (I can't mark it as resolved) I thought it was a typo. 😉